### PR TITLE
feat: add how to create a tax plugin

### PR DESCRIPTION
**Pull request by brent-hoover**
Signed-off-by: Brent Hoover <brent@thebuddhalodge.com>

Updates from old docs:

Changed title to correct format
Removed non-existent links
Linked to existing example tax plugin
Removed references to React component
Corrected call signature for `calculateOrderTaxes` (includes cart)
Double-checked that return payload is still the same

### CODE REVIEW

File: how-tos/create-a-tax-plugin.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+
+# How To: Add a Tax Service plugin
+
+## Prerequisites
+- Understanding how to create and integrate a plugin
```
Comment (MohanNarayana) [inline how-tos/create-a-tax-plugin.md]: Maybe we can add a link to create plugin here.

File: how-tos/create-a-tax-plugin.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+```js
+{
+  _id: "123", // Generate a random unique ID if you don't have one
+  jurisdictionId: "123", // Optionally provide an ID for the jurisdiction this tax is for. Not currently used by core.
+  sourcing: "destination", // Either "destination" or "origin" depending on which address triggered this tax
+  tax: 1, // Total amount due for this type of tax only, in `order.currencyCode`
+  taxableAmount: 1, // Total deemed taxable for this type of tax only, in `order.currencyCode`
+  taxName: "CA Sales Tax", // A human-readable string for showing to operators and customers in the UI
+  taxRate: 3.5, // The tax rate for this type of tax, used for this calculation
```
Comment (MohanNarayana) [inline how-tos/create-a-tax-plugin.md]: Also mention that the value is a percentage.

**Review by MohanNarayana — CHANGES_REQUESTED** [review]

**Review by delagroove — APPROVED** [review]

**Comment by brent-hoover** [comment]
@MohanNarayana Changes from c/r made

**Comment by brent-hoover** [comment]
replaced by #67

**Pull request closed without merging**